specification.md

```diff
@@ -5157,43 +5225,37 @@ Workflow constants should not have access to [Workflow secrets definitions](#Wor
 
 ### Workflow Secrets
 
-Secrets allow you access sensitive information, such as passwords, OAuth tokens, ssh keys, etc
-inside your [Workflow Expressions](#Workflow-Expressions). 
+Secrets allow you access sensitive information, such as passwords, OAuth tokens, ssh keys, etc.
+They can be used in [Workflow Expressions](#Workflow-Expressions).
 
 You can define the names of secrets via the [Workflow top-level "secrets" property](#Workflow-Definition-Structure), 
 for example:
 
 ```json
-"secrets": ["MY_PASSWORD", "MY_STORAGE_KEY", "MY_ACCOUNT"]
+"secrets": ["MY_ACCOUNT", "MY_PASSWORD", "urn:example:tenant:accessToken"]
```

Comment:
cant this be done as a separate pr for secrets and not ghost added to this one please? if you would like this change then lets just create a separate pr for it as idk what its got to do w/ expressions

Reply:
secrets can be an array of strings. Maybe I missed something, but I think it can be an arbitrary string, no? A urn is just a string, so nothing is being ghost-added